### ISSUE TITLE
[precise-gce] Twisted fails to install with the clang build

### ISSUE THREAD

**Issue by sobomax** [travis-ci/travis-ci#5348]
Hi, we've noticed that for some reason the Travis/clang environment now fails to install Twisted. But it still works fine in the gcc environment. We believe this is connected to the migration of environments to the new infrastructure, our last clean build was 22 days ago.

Failed build here:

https://travis-ci.org/sippy/rtpproxy/jobs/98179011

Good one using gcc:

https://travis-ci.org/sippy/rtpproxy/jobs/98179012

**Comment by BanzaiMan** [comment]
Might be a problem with `$PATH`. Could you try the workaround suggested in #5326?

**Comment by sobomax** [comment]
Yes, we've been able to workaround it by adding /usr/local/clang-3.4/bin into a PATH explicitly before running pip. Not sure why it started to fail though.
